NitroxClient/GameLogic/PlayerDeath.cs

```diff
@@ -0,0 +1,22 @@
+﻿using NitroxClient.Communication;
+using NitroxModel.Packets;
+using UnityEngine;
+
+namespace NitroxClient.GameLogic
+{
+    public class PlayerDeath
```

Comment:
Can we change this to PlayerEvents or something similar?  Just so we can tuck similar logic in with it.

Reply:
I thought about doing this initially but strayed away because I didn't want a "god" player event class. I felt like there could be future enhancements made to player death alone that may not necessarily transition into other player-driven events.